From b977b287f61fea48ecd6251d54a26334213b7ec6 Mon Sep 17 00:00:00 2001
From: Diego Biurrun <diego@biurrun.de>
Date: Fri, 15 Aug 2014 22:13:14 +0200
Subject: [PATCH 11/37] vsrc_movie: Avoid a variable indirection in
 movie_get_frame()

This avoids an unused variable warning with MSVC since the variable is
only used in a debug mode printf statement.
---
 libavfilter/vsrc_movie.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/libavfilter/vsrc_movie.c b/libavfilter/vsrc_movie.c
index 8ebdd06..1ee0f16 100644
--- a/libavfilter/vsrc_movie.c
+++ b/libavfilter/vsrc_movie.c
@@ -206,7 +206,6 @@ static int movie_get_frame(AVFilterLink *outlink)
     MovieContext *movie = outlink->src->priv;
     AVPacket pkt;
     int ret, frame_decoded;
-    AVStream av_unused *st = movie->format_ctx->streams[movie->stream_index];
 
     if (movie->is_done == 1)
         return 0;
@@ -226,7 +225,8 @@ static int movie_get_frame(AVFilterLink *outlink)
                 av_dlog(outlink->src,
                         "movie_get_frame(): file:'%s' pts:%"PRId64" time:%f aspect:%d/%d\n",
                         movie->file_name, movie->frame->pts,
-                        (double)movie->frame->pts * av_q2d(st->time_base),
+                        (double)movie->frame->pts *
+                        av_q2d(movie->format_ctx->streams[movie->stream_index]),
                         movie->frame->sample_aspect_ratio.num,
                         movie->frame->sample_aspect_ratio.den);
                 // We got it. Free the packet since we are returning
-- 
1.9.1

